### ISSUE TITLE
Problems on small screens with left sidebar

### ISSUE THREAD

**Issue by deakonizer** [modxcms/revolution#16029]
### Summary
On a small Laptop screen you cant see the whole Reports. Only the Site Schedule and you can't scroll down. Also the left Bar takes spaces on small screens. 

### Why is it needed?
You cant see the whole Logs and Reports. Here is a screenshot:

![afbeelding](https://user-images.githubusercontent.com/312944/152353833-df97f5c4-12eb-4145-af97-5e46a39e33df.png)

### Suggested solution(s)
The new Theme looks pretty cool and fresh, but, bring the Sidebar back to top. Maybe the User have the choice per Toggle (left or top), or maybe you have the choice for different Manager Themes (Radio Button)

Mfg
Marco

**Comment by Jako** [comment]
You are right: The sidebar menu needs some additional work. The position of the subsubnav has to be calculated on size of the screen. The leftbar can be hidden with the arrow below the MODX logo.

**Comment by Ruslan-Aleev** [comment]
> Maybe the User have the choice per Toggle (left or top), or maybe you have the choice for different Manager Themes (Radio Button)

This is too global changes to make (I personally don't like the menu on the left either).
If you have time and desire - we will be glad to receive PR from you :)

**Comment by Mark-H** [comment]
Fixing the subsubnav with priority would definitely be useful, but I just want to state for the record that I seriously doubt we'd accept a massive change to the UI at this point in the release cycle. We're two weeks from the planned stable release, just not worth the risk breaking more things so short to the deadline.

**Comment by Jako** [comment]
The subsubnav position is a bug. And there is another issue that has to be fixed too working on the bug: the navigation bar has to be reloaded/recalculated after resizing the window.

**Comment by smg6511** [comment]
As @Jako said, there's definitely a bug there in the calculations that needs to be fixed. As to the left menu in general, I've never been a fan of it (from a usability standpoint), but have also known that as @Mark-H said it would be a major undertaking to change it — one that would _almost_ put us back at "square one" UI-wise. If one had a lot of time on their hands however, an optional manager theme could be developed, which would be a first and be great but a whole lot of work ;-)

**Comment by Jako** [comment]
I am working on this. It is fine on large screens but the menu currently has issues on mobile screens (between 640px and 960px width).
